modules/v2x/fusion/configs/ft_config_tools.h

```diff
@@ -0,0 +1,68 @@
+// Copyright 2016 Baidu Inc. All Rights Reserved.
+// @author: Jary (lizheng23@baidu.com)
+// @file: ft_config_manager.cc
+// @brief:
+#ifndef FT_CONFIG_TOOLS_H_
+#define FT_CONFIG_TOOLS_H_
+
+#include <fcntl.h>
+#include <google/protobuf/io/zero_copy_stream_impl.h>
+#include <google/protobuf/text_format.h>
+#include <stdlib.h>
+#include <fstream>
+#include <iostream>
+#include <string>
+
+#include "modules/v2x/fusion/configs/fusion_tracker_gflags.h"
+namespace v2x {
+namespace ft {
+namespace tools {
+
+// @brief load protobuf(TXT) data from file.
+template <typename T>
+bool ParseProtobufFromFile(const std::string &file_name, T *pb) {
```

Comment:
Reuse what was already provided by `cyber` and `modules/common`

Reply:
Agree. Use `GetProtoFromFile` defined in `cyber/common/file.h`